src/django/api/management/commands/reprocess_geocode_failures.py

```diff
@@ -0,0 +1,78 @@
+from django.core.management.base import BaseCommand
+from django.db import transaction
+from django.db.models import Count
+
+from api.constants import ProcessingAction
+from api.models import FacilityList, FacilityListItem
+from api.processing import (parse_facility_list_item,
+                            geocode_facility_list_item,
+                            match_facility_list_items,
+                            save_match_details)
+
+LINE_ITEM_ACTIONS = {
+    ProcessingAction.PARSE: parse_facility_list_item,
+    ProcessingAction.GEOCODE: geocode_facility_list_item,
+}
+
+LIST_ACTIONS = set([ProcessingAction.MATCH])
```

Comment:
Could this just be `{ProcessingAction.MATCH}`?

Reply:
Oops. These constants were copy pasted from a previous management command and are not even used. Will remove.